homeassistant/components/androidtv/media_player.py

```diff
@@ -187,6 +187,7 @@ def _adb_decorator(func):
         @functools.wraps(func)
         async def _adb_exception_catcher(self, *args, **kwargs):
             """Call an ADB-related method and catch exceptions."""
+            # pylint: disable=protected-access
```

Comment:
Is pylint not able to detect that the decorator is decorating a method?

Reply:
Unfortunately not. That's similar to Pylance btw if the decorator function would have type annotations. `adb_decorator` is a normal function outside any class scope, so `self` doesn't have any special meaning here.

One could argue to exclude `self._xxx` and `cls._xxx` by default, in fact that is what I originally thought happened which is why I didn't recognize the regression, but for now the change just restores the behavior prior to the `2.13.0` update.

Reply:
> One could argue to exclude `self._xxx` and `cls._xxx` by default

The problem then of course becomes what to do if the first argument is named something different, like `device` for the next change (in `decora/light.py`).

In the end, there might not be a good solution here, other then only calling public attributes and methods or adding a `disable=protected-access` comment.